tangelo/linq/tests/test_gates.py

```diff
@@ -52,6 +52,11 @@ def test_some_gates_inverse(self):
         H_gate_inverse = Gate("H", 2)
         self.assertEqual(H_gate.inverse().__str__(), H_gate_inverse.__str__())
 
+        # Create a SWAP gate acting on qubits 1 and 2
+        H_gate = Gate("SWAP", [1, 2])
+        H_gate_inverse = Gate("SWAP", [1, 2])
+        self.assertEqual(H_gate.inverse().__str__(), H_gate_inverse.__str__())
```

Comment:
I don't know how old these tests are, but we should use Gate equality operators (==) to compare gates. Would you please quickly rewrite those four?
(also calling your SWAP gate "H_gate" is not the best.)

I think they could look something like that:

```
H_gate = Gate(...)
self.assertEqual(H_gate.inverse(), H_gate)

RZ_gate = Gate(...)
self.assertEqual(RZ_gate.inverse(), Gate('RZ', .... <change parameter>))
```